packages/gamepads/example/.gitignore

```diff
@@ -42,3 +42,12 @@ app.*.map.json
 /android/app/debug
 /android/app/profile
 /android/app/release
+
+
+/test
```

Comment:
feels like we shouldn't preemptively ignore tests just because we don't have any _yet_ :P

Reply:
Is it common to test example project's?

Reply:
Yeah, let's not introduce tests in examples 😅
I think this one is good, because you often run flutter create and it will create this directory, but you don't want to commit it.

Reply:
ah sorry I didn't see it was an example folder. thought it was for the whole repo